content/blog/getting-started-with-hpe-greenlake-api-for-backup-and-recovery.md

```diff
@@ -0,0 +1,516 @@
+
+### What about the components in HPE GreenLake Backup and Recovery that are not mentioned above?
+
+There are other resources exist in the HPE GreenLake for Backup and Recovery that are not mentioned above; however, they are available in the user-interface for HPE GreenLake for Backup and Recovery. A couple of those resources are protection policy, and protection group. The protection group is used to consolidate multiple number of assets with a particular protection policy. On the other hand, the protection-policy is a resource to consolidate the schedule of protection-jobs and the flow of the recovery points at different tiers of protection-store. Together, both protection policy and protection group deliver the management of the Recovery Point Objective for data protection using HPE GreenLake for Backup and Recovery.
+
+![Protection policy for vmware schedule](/img/protection-policy-ui-for-vmware-scheduling.png)
+
+*The above figure display resources for schedule and the flow of the recovery-points part of the protection policy.*
+
+Additionally, there is also a resource called a protection job that is not shown in the user-interface; however, it is required to perform the operation of data protection. You will need to manipulate protection jobs to create a protection or to recover an existing recovery point. Additionally, the protection-job is the resource that can be manipulated to suspend or to resume a protection schedule. 
+
+![Protection jobs for scheduling and protection tiers](/img/protection-jobs-ui.png)
+
+*The above figure displays the representation of protection jobs which are the resources available after a protection policy applied to a protection group of many assets or a single asset.*
+
+Beyond protection-jobs, there are recovery points (also known as copies). These are resources that used to represent a copy that is inside the protection store, and they are used to perform the recovery. Note that even though there are three different categories of recovery-points that are available in the protection policy, there are only two different resources. The first resource is called snapshots to represent copies that exist inside the storage array. The second resource is called backups to represent copies that exist inside the on-premises protection store such as Protection Store Gateway, and copies that exist inside the cloud protection store. The cloud protection store is repository that sits in the cloud public providers that are managed by HPE, and these can exist either in AWS or Azure. All of these recovery points exist in relation to the assets, where the recovery can be initiated using the HPE GreenLake API.
```

Comment:
--> These are resources that are used to represent a copy...

--> Microsoft Azure